### PR TITLE
MOD-6534: Missing values indexing - Add support to indexing pipeline

### PR DESCRIPTION
**Pull request by nafraf**
**Description**

A clear and concise description of what the PR is solving, including:
1. The current state briefly
2. What is the change
3. Adding the outcome (changed state)

**Which issues this PR fixes**
1. MOD-6534

**Main objects this PR modified**
1. ...
2. ...

**Mark if applicable**

- [ ] This PR introduces API changes
- [ ] This PR introduces serialization changes

### CODE REVIEW

File: src/redisearch_api.c [hunk 1]
```diff
@@ -44,6 +44,7 @@ RefManager* RediSearch_CreateIndex(const char* name, const RSIndexOptions* optio
   IndexSpec* spec = NewIndexSpec(name);
   StrongRef ref = StrongRef_New(spec, (RefManager_Free)IndexSpec_Free);
   IndexSpec_MakeKeyless(spec);
+  spec->missingFieldDict = dictCreate(&dictTypeHeapStrings, NULL);
```
Comment (raz-mon) [inline src/redisearch_api.c:47]: Add to `IndexSpec_MakeKeyless()`

File: src/fork_gc.c [hunk 2]
```diff
@@ -1078,6 +1097,69 @@ static FGCError FGC_parentHandleTags(ForkGC *gc) {
...
+    }
+  }
+
+cleanup:
+
+  if (sp) {
+    RedisSearchCtx_UnlockSpec(sctx);
+    StrongRef_Release(spec_ref);
+  }
+
```
Comment (raz-mon) [inline src/fork_gc.c:1154]: Free `fieldName`

**Review by raz-mon — COMMENTED** [review]
Great :100
See few comments

File: src/fork_gc.c (hunk 2 above)
Comment (nafraf) [reply src/fork_gc.c]: OK. Done

File: src/redisearch_api.c (hunk 1 above)
Comment (nafraf) [reply src/redisearch_api.c]: Done.

**Pull request merged**